src/styles/util/_helpers.scss

```diff
@@ -23,6 +23,32 @@
   }
 }
 
+/*============= Flex Layout ================*/
+.flex-row {
+  display: flex;
+  flex-direction: row;
+}
+
+.flex-column {
+  display: flex;
+  flex-direction: column;
+}
+
+.center-both {
+  justify-content: center;
+  align-items: center;
+}
```

Comment:
actually some styles like this already exist via a library i installed, svbstrate. see below:
```css
.f { display: flex }
.fw { flex-wrap: wrap }
.ais { align-items: flex-start }
.aie { align-items: flex-end }
.aic { align-items: center }
.aib { align-items: baseline }
.jcs { justify-content: flex-start }
.jce { justify-content: flex-end }
.jcc { justify-content: center }
.jcb { justify-content: space-between }
.fa {
  flex: 1 1 auto;
  min-width: 0;
  min-height: 0;
}

```

so you can add combinations of these classes to html elements to achieve the desired flexbox effect

more helpers included with the library can be found here, sorry for not mentioning it sooner!: https://github.com/estrattonbailey/svbstrate/tree/master/src/lib

Reply:
My bad, I should have checked for this. I'll remove the helpers I wrote and replace the class names with these shorter ones